presto-native-execution/README.md

```diff
@@ -1,4 +1,4 @@
-# presto.cpp
+# presto native execution - running Presto workers in native binary
```

Comment:
We decided to use `Prestissimo` instead of presto native execution.
`Prestissimo - Presto worker implementation using Velox` is probably apt.

Reply:
> Prestissimo - C++ Presto worker implementation using Velox

That makes sense. Will update the text accordingly.